db/migrate/20210720042422_drop_custom_profile_fields.rb

```diff
@@ -0,0 +1,12 @@
+class DropCustomProfileFields < ActiveRecord::Migration[6.1]
+  disable_ddl_transaction!
+
+  def change
```

Comment:
I didn't bother to make this migration fully reversible by providing a block for `custom_profile_fields`. This is the second time we're removing this feature, I hope it doesn't come back again 😅 

I still tried to keep it somewhat robust, so used `if_exists` which has been around for `drop_table` since 2015 and was added for `remove_index` with Rails 6.1.